esp/bindings/http/platform/httpbinding.cpp

```diff
@@ -412,6 +418,71 @@ void EspHttpBinding::readUnrestrictedResources(const char* resources)
     }
 }
 
+bool EspHttpBinding::canRedirectAfterAuth(const char* url)
```

Comment:
Perhaps a single line comment before each new method to explain what/why it does ?

Reply:
const method